polylogue/cli.py

```
@@ -785,6 +1070,109 @@ def run_import_claude_code(args: argparse.Namespace, env: CommandEnv) -> None:
         return
 
     summarize_import(ui, "Claude Code Import", [result])
+    if getattr(args, "to_clipboard", False):
+        copy_import_to_clipboard(ui, [result])
+
+
+def run_automation_cli(args: argparse.Namespace, env: CommandEnv) -> None:  # noqa: D401
```

Ruff (0.15.9)

[warning] Unused function argument: `env`

(ARG001)